src/jordan_wigner.rs

```diff
@@ -0,0 +1,118 @@
+use crate::fermion_operator::FermionOperator;
+use numpy::Complex64;
+use pyo3::prelude::*;
+use std::collections::HashMap;
+
+type SparseLabel = String;
+type SparseIndices = Vec<usize>;
+type SparseCoeff = Complex64;
+type SparseListEntry = (SparseLabel, SparseIndices, SparseCoeff);
+type SparseList = Vec<SparseListEntry>;
+
+/// Jordan–Wigner map of a FermionOperator.
+///
+/// Returns (sparse_list, num_qubits) where sparse_list is used to construct a Sparse Pauli Operator in Qiskit.
+#[pyfunction]
+pub fn jordan_wigner(op: &FermionOperator, norb: usize) -> PyResult<(SparseList, usize)> {
```

Comment:
Let's rename this function `jordan_wigner_qiskit` because its output is specifically made for Qiskit. I would like to eventually add a native QubitOperator class to ffsim, and want to reserve `jordan_wigner` for the function that outputs that class.